python/lsst/ip/diffim/imageMapReduce.py

```diff
@@ -787,7 +787,7 @@ def _plotBoxGrid(self, boxes, bbox, **kwargs):
         Parameters
         ----------
         boxes : `list`
-            a list of `afwGeom.BoundingBox`es
+            a list of `lsst.afw.geom.BoundingBox`es
```

Comment:
Shouldn't this be `lsst.geom.BoundingBox` ?

Reply:
I was meant to change to to `lsst.geom.Box2I`.